### DISCUSSION TITLE
Construction Feature and Boxes

### DISCUSSION THREAD

**Discussion by coltKeegan** [official-antistasi-community/A3-Antistasi#3100] [Q&A] [answered]
Hello there, I require some help regarding the construction feature that's there on the mod, through which we can build and place buildings, bunkers, trenches etc etc... 
I've tried to purchase a small and big construction box through the garage (role set to teamleader/engineer), however when I enter the 'building placement mode' through the box, the list of buildings appears for a split second then disappears. Morever there's no prompt to exit that placement mode anyhow, and the camera just stays in that drone shot viewmode, I'm not sure if this is a bug or I'm missing something. In any case, if I'm missing something then I'll be grateful if someone can walk me through the correct process.

**Comment by jaj22** [comment, marked as answer]
Mod conflict maybe? Test with just Antistasi.
